quarkus/extensions/pom.xml

```diff
@@ -89,15 +112,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <environmentVariables>
-                        <KEYCLOAK_CAMEL_CASE_SCOPE_CAMEL_CASE_PROP>foobar</KEYCLOAK_CAMEL_CASE_SCOPE_CAMEL_CASE_PROP>
```

Comment:
This is used by the config test that Dmitry added, not sure why it doesn't fail without it. Kinda not related to this PR though?

Reply:
It is not, but I did see any difference after removing it. So I just removed it.